internal/build/phase.go

```diff
@@ -3,31 +3,27 @@ package build
 import (
 	"context"
 	"io"
-	"os"
-	"runtime"
-	"sync"
 
 	"github.com/docker/docker/api/types"
 	dcontainer "github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/client"
 	"github.com/pkg/errors"
 
-	"github.com/buildpacks/pack/internal/archive"
 	"github.com/buildpacks/pack/internal/container"
 )
 
 type Phase struct {
-	name        string
-	infoWriter  io.Writer
-	errorWriter io.Writer
-	docker      client.CommonAPIClient
-	ctrConf     *dcontainer.Config
-	hostConf    *dcontainer.HostConfig
-	ctr         dcontainer.ContainerCreateCreatedBody
-	uid, gid    int
-	appPath     string
-	appOnce     *sync.Once
```

Comment:
Nice job doing away with this... this is the part that was giving me trouble before :)

Reply:
There is a "risk" now that IF you don't run `detect` or `create` FIRST that the app contents won't be there but I didn't see that as a reason why we need to _try_ to copy the app during every phase.